ui/pages/confirmations/confirmation/alerts/useAddEthereumChainAlerts.ts

```diff
@@ -0,0 +1,61 @@
+import { ApprovalRequest } from '@metamask/approval-controller';
+import { ApprovalType } from '@metamask/controller-utils';
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Alert } from '../../../../ducks/confirm-alerts/confirm-alerts';
+import { AlertActionKey } from '../../../../components/app/confirm/info/row/constants';
+import { Severity } from '../../../../helpers/constants/design-system';
+import { getMemoizedUnapprovedConfirmations } from '../../../../selectors';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const VALIDATED_APPROVAL_TYPES = [
+  ApprovalType.AddEthereumChain,
+  ApprovalType.SwitchEthereumChain,
```

Comment:
Is this for a future PR as we haven't enabled the switch confirmation yet?

Reply:
Yes